src/engraving/layout/layoutoptions.h

```diff
@@ -43,39 +39,10 @@ struct LayoutOptions
 {
```

Comment:
Do we still need this struct?

Reply:
I'm thinking of moving fields from Score into it, which are related to layout settings, in order to structure and simplify Score